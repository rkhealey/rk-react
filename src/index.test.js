import _ from 'lodash';
import initStoryshots from '@storybook/addon-storyshots';

import index from './';

initStoryshots();

describe('/index', () => {
  it('should export components', () => {
    expect(_.isFunction(index.Card)).toEqual(true);
  });
});
