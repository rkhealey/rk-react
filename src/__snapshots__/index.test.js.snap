// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button button block 1`] = `
<button
  className="sc-bxivhb hmsGEH"
  onClick={[Function]}
  type="button"
>
  click me
</button>
`;

exports[`Storyshots Button button default 1`] = `
<button
  className="sc-bxivhb nYfzy"
  onClick={[Function]}
  type="button"
>
  click me
</button>
`;

exports[`Storyshots Button button floating 1`] = `
<button
  className="sc-bxivhb irTQGt"
  onClick={[Function]}
  type="button"
>
  click me
</button>
`;

exports[`Storyshots Button button invisible 1`] = `
<button
  className="sc-bxivhb elmphq"
  onClick={[Function]}
  type="button"
>
  click me
</button>
`;

exports[`Storyshots Card card default 1`] = `
<div
  className="sc-ifAKCX fAWOVe"
>
  <h3>
    Iâ€™m the card component
  </h3>
</div>
`;

exports[`Storyshots Grid grid default 1`] = `
<div
  className="sc-htpNat dlGndh"
>
  <div
    className="sc-bwzfXH kgpiyF"
    spacing="space-between"
  >
    <div
      className="sc-bdVaJa dgmonm"
    >
      <div
        className="sc-htoDjs jwxkCT"
      >
        Col 1
      </div>
    </div>
    <div
      className="sc-bdVaJa dgmonm"
    >
      <div
        className="sc-htoDjs jwxkCT"
      >
        Col 2
      </div>
    </div>
    <div
      className="sc-bdVaJa dgmonm"
    >
      <div
        className="sc-htoDjs jwxkCT"
      >
        Col 3
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Grid grid even columns 1`] = `
<div
  className="sc-htpNat dlGndh"
>
  <div
    className="sc-bwzfXH kgpiyF"
    spacing="space-between"
  >
    <div
      className="sc-bdVaJa Ablql"
    >
      <div
        className="sc-htoDjs jwxkCT"
      >
        Col 1
      </div>
    </div>
    <div
      className="sc-bdVaJa Ablql"
    >
      <div
        className="sc-htoDjs jwxkCT"
      >
        Col 2
      </div>
    </div>
    <div
      className="sc-bdVaJa Ablql"
    >
      <div
        className="sc-htoDjs jwxkCT"
      >
        Col 3
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Icon icon default 1`] = `
<i
  className="material-icons sc-EHOje cBLLRW"
  color={null}
>
  face
</i>
`;

exports[`Storyshots Icon icon with button 1`] = `
<button
  className="sc-bxivhb eLWyNB"
  onClick={[Function]}
  type="button"
>
  <i
    className="material-icons sc-EHOje cBLLRW"
    color={null}
  >
    face
  </i>
</button>
`;

exports[`Storyshots Icon icon with color 1`] = `
<i
  className="material-icons sc-EHOje krNhiV"
  color="purple"
>
  face
</i>
`;

exports[`Storyshots Modal modal default 1`] = `
<div>
  <div />
  <button
    className="sc-bxivhb iFIPVN"
    onClick={[Function]}
    type="button"
  >
    Open Modal
  </button>
</div>
`;

exports[`Storyshots Transition Fade 1`] = `
<div>
  <div>
    <button
      className="sc-bxivhb iFIPVN"
      onClick={[Function]}
      type="button"
    >
      Toggle
    </button>
  </div>
</div>
`;
